### PR TITLE
Make ChangeLog format compatible with Markdown/reStructuredText

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

File: scripts/release [hunk 1]
```diff
@@ -48,7 +48,7 @@ echo "Using tag reference: ${annotation} as release tag.\n"
 echo ${annotation} > ${VERSION_FILE}
 
 mv ChangeLog.md ChangeLog.md.old
-echo "# Lepidopter v${annotation} [`date +%Y-%m-%d`]\n" > ChangeLog.md
+echo "Lepidopter v${annotation} [`date +%Y-%m-%d`]\n" > ChangeLog.md
```
Comment (hellais) [inline scripts/release]: You probably also want to print the `----` here.

File: ChangeLog.md [hunk 2]
```diff
@@ -20,8 +22,10 @@
 
 v0.2.1-alpha
 ------------
-* Cleanup ooniprobe logrotated files
+
+- Cleanup ooniprobe logrotated files
 
 v0.2.0-alpha
```
Comment (hellais) [inline ChangeLog.md]: You should add the date to these previous two releases as well

File: scripts/release (hunk 1 above)
Comment (anadahz) [reply scripts/release]: Fixed in 8df2c96

File: ChangeLog.md (hunk 2 above)
Comment (anadahz) [reply ChangeLog.md]: Fixed in 8df2c96

**Pull request merged**